staging/src/k8s.io/api/testdata/HEAD/autoscaling.v2.HorizontalPodAutoscaler.json

```diff
@@ -1,282 +0,0 @@
-{
```

Comment:
this being missing was a bug... added autoscaling v2 types in https://github.com/kubernetes/kubernetes/pull/108055 and updated the fixtures to match what we shipped in 1.23.0